README.md

```diff
@@ -49,6 +49,9 @@ has slowed down enough.
 
   // Remove hoverintent listeners
   hoverListener.remove();
+
+  // Remove hoverintent and fire the out handler
+  hoverListener.remove(true);
```

Comment:
Wasn't sure this was the right place for it, but wanted to show a real world usage. If this is intended as a snippet that can be copy/pasted, might be worth moving this to a different location.